docs/BR.md

```diff
@@ -168,32 +162,34 @@ The following Certificate Policy identifiers are reserved for use by CAs as an o
 The CA/Browser Forum is a voluntary organization of Certification Authorities and suppliers of Internet browser and other relying-party software applications.
 
 ### 1.3.1 Certification Authorities
-Certification Authority (CA) is defined in Section 1.6. Current CA Members of the CA/Browser Forum are listed here: <https://cabforum.org/members>.
+Certification Authority (CA) is defined in [Section 1.6](#definitions-and-acronyms). Current CA Members of the CA/Browser Forum are listed here: <https://cabforum.org/members>.
```

Comment:
https://pandoc.org/MANUAL.html#headings-and-sections captures how Pandoc currently generates the identifiers. Explicitly specifying identifiers in headings is one option (explained a little in https://github.com/jgm/pandoc/issues/2821#issuecomment-269847161 re: HTML4 limitations). I didn't push that too far, but happy to revisit based on folks' thoughts.

Reply:
I just went ahead and did it, because you're right, it's clearer and easier to make sure cross-references are correct. Once https://github.com/cabforum/build-guidelines-action/pull/5 lands and a new RC is cut, I'll uprev this.